### PR TITLE
Christy/week1/3 saturday/homework/exercise1

### PR DESCRIPTION
**Pull request by christyph**

### CODE REVIEW

File: week1/2-thursday/homework/exercise1/answer1.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+///////// exercise 1 //////////
+
+function hello(name) {
+    // put your code here
+    return "Hello, " + name + '!';
+
+
+  }
+console.log(hello('Mustache'));
```
Comment (DanStough) [inline week1/2-thursday/homework/exercise1/answer1.js:9]: I think this file is a duplicate.

**Review by DanStough — CHANGES_REQUESTED** [review]

**Comment by christyph** [comment]
I see, you're right! it seems the answer was posted onto both Thursday and Saturday folders by mistake

File: week1/2-thursday/homework/exercise1/answer1.js (hunk 1 above)
Comment (christyph) [reply week1/2-thursday/homework/exercise1/answer1.js:9]: you're right, I see that I posted the answer onto both Thursday and Saturday twice

Comment (DanStough) [reply week1/2-thursday/homework/exercise1/answer1.js:9]: BTW, when you want to remove a file that is already added to git, you would can use `git rm <filename>`.

**Comment by DanStough** [comment]
Closing for now unless we want to fix the duplicate file. If that's the case we can open the PR again.

**Pull request closed without merging**